packages/web/src/djitsu/components/notebook-header/index.js

```diff
@@ -291,6 +297,57 @@ export const NotebookHeader = (props) => {
   //   { hasMain, notebook, userProfile, notebookId, isOwner }
   // )
 
+  const fallbackCopyTextToClipboard = (text) => {
+    const textArea = document.createElement('textarea')
+    textArea.value = text
+
+    // Avoid scrolling to bottom
+    textArea.style.top = '0'
+    textArea.style.left = '0'
+    textArea.style.position = 'fixed'
+
+    document.body.appendChild(textArea)
+    textArea.focus()
+    textArea.select()
+
+    try {
+      const successful = document.execCommand('copy')
+      successful
+        ? message.success('Copied published location')
+        : message.error('Failed to copy')
+    } catch (err) {
+      message.error('Fallback: Oops, unable to copy', err)
+    }
+
+    document.body.removeChild(textArea)
+  }
+
+  const copyTextToClipboard = (text) => {
+    if (!navigator.clipboard) {
+      fallbackCopyTextToClipboard(text)
+      return
+    }
+    navigator.clipboard.writeText(text).then(
+      function () {
+        message.success('Import address copied to clipboard')
+      },
+      function (err) {
+        message.error('Import address copied to clipboard', err)
+      }
+    )
+  }
+
+  const initCopy = () => {
+    let exportsAsString = ''
```

Comment:
I'd prepare the string in advanced so it can be shown to the user - this can be a large blob of text on some occasions and not necessarily what the user expects.